### PR TITLE
v1.1 release notes

### LINKED ISSUE #1967: Update docs repo for 1.1 release
1. Add `v1.2` directory
2. Update `_config.yml` to point identify `v1.1` as `stable` and `v1.2` as `dev`.
3. Add `1.1` to `Releases` section of sidenav.

### LINKED ISSUE #1969: Summarize all user-facing changes in 1.1
For release notes.

### PR DESCRIPTION
**Pull request by jseldess**
Fixes #1969 
Fixes #1967

### CODE REVIEW

File: releases/v1.1.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+[`DROP USER`](../v1.1/drop-user.html) | Use this new statement to remove one or more SQL users. This features is equivalent to the existing [`cockroach user rm`](../v1.1/create-and-manage-users.html) command.
+[`DROP DATABASE`](../v1.1/drop-database.html) | Use the new `CASCADE` option to drop all tables and views in a database as well as all objects (such as [constraints](constraints.html) and [views](views.html)) that depend on those tables. This option does not list objects it drops, so should be used cautiously.
+[`INSERT`](../v1.1/insert.html#update-values-on-conflict) | Use a `WHERE` clause with `ON CONFLICT` to conditionally apply updates that conflict a Unique constraint on one or more columns.
+
+### SQL Types
+
+Type | Description
+-----|------------
+[`ARRAY`](../v1.1/array.html) | Use this new data type to store 1-dimensional, 1-indexed, homogenous arrays of any non-array data type.
+[`UUIC`](../v1.1/uuid.html) | Use this new data type to store 128-bit values that are globally unique. This type is recommended for [auto-generating unique row IDs](../v1.1/uuid.html#create-a-table-with-auto-generated-unique-row-ids).
```
Comment (jtescher) [inline releases/v1.1.md]: Should this be [`UUID`](..) instead of [`UUIC`](..)?

Comment (jseldess) [reply releases/v1.1.md]: Yep, good catch. Thanks, @jtescher.

File: releases/v1.1.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+---
+title: What&#39;s New in v1.1
+toc: false
+summary: Additions and changes in CockroachDB version v1.1
+redirect_from: /v1.0.html
+---
+
+## DATE
+
+With the release of CockroachDB v1.1, we’ve included support for faster migrations from legacy databases for developers, simplified cluster management for operators, and improved performance in real-world environments. Read about the implications of these changes in [this blog post](https://www.cockroachlabs.com/blog/cockroachdb-1-0-release/), checkout a more granular [summary of new SQL, CLI, and Admin UI features](#summary) below, and then [upgrade to CockroachDB v1.1](../v1.1/upgrade-cockroach-version.html).
```
Comment (bdarnell) [inline releases/v1.1.md]: "Legacy" is too harsh.

Don't forget to update the blog link

File: releases/v1.1.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- [Cockroach Commands](#cockroach-commands)
+- [Admin UI](#admin-ui)
+- [Known Limitations](#known-limitations)
+
+### SQL Statements
+
+Statement | Description
+----------|------------
+[`IMPORT`](../v1.1/import.html) | Use this new statement to import an entire table's data into a running cluster by loading CSV files.
+[`SHOW QUERIES`](../v1.1/show-queries.html)<br>[`CANCEL QUERY`](../v1.1/cancel-query.html) | Use these new statements to monitor the progress of active queries, and, if necesssry, cancel long-running queries causing unwanted resource consumption.
```
Comment (bdarnell) [inline releases/v1.1.md]: "necessary"

File: releases/v1.1.md (hunk 2 above)
Comment (jseldess) [reply releases/v1.1.md]: Changed "legacy" to "other". Thanks for the reminder about the link. I don't have it yet, but I'll be sure to update this once I do.

File: releases/v1.1.md (hunk 3 above)
Comment (jseldess) [reply releases/v1.1.md]: Done.

**Comment by jseldess** [comment]
@jess-edwards, once you have the final url to the 1.1 blog post, please just post it here so I'm sure to update the release notes.

**Comment by jess-edwards** [comment]
@jseldess The blog will be at: 
https://www.cockroachlabs.com/blog/cockroachdb-1dot1/

**Comment by jseldess** [comment]
Thanks, @jess-edwards!

File: releases/v1.1.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v1.1.0.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+## Summary
+
+This section summarizes the most significant, user-facing changes included in v1.1. For a complete list of changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases.
```
Comment (bdarnell) [inline releases/v1.1.md]: We need to call out all the backwards-incompatible changes here too. (I'd put this before any discussion of new features). We should also call out somewhere that upgrading from 1.0.x to 1.1 requires a different procedure from upgrades within the 1.0.x series.

**Comment by jess-edwards** [comment]
@jseldess There's a URL missing on "cloud deployment tutorials". It links to nothing.

**Comment by jseldess** [comment]
Thanks, @jess-edwards. Fixed.

File: releases/v1.1.md (hunk 4 above)
Comment (jseldess) [reply releases/v1.1.md]: @bdarnell, thank you for calling that out! I've added a list of backwards-incompatible changes and note that the upgrade process is different.

**Comment by jseldess** [comment]
TC failed on the blog link, but I know the link is accurate and live now, so I'm overriding and merging.

File: _config.yml [hunk 5]
```diff
@@ -18,19 +18,24 @@ redcarpet:
   - with_toc_data
 
 versions:
-  stable: v1.0
-  dev: v1.1
+  stable: v1.1
+  # dev: v1.2
```
Comment (tamird) [inline _config.yml]: note that without _any_ dev docs, the various links in the cockroach repo are now broken, causing tests to fail

Comment (jseldess) [reply _config.yml]: Yuck, thanks, @tamird. Something to remember for next time. Alex merged a fix just a bit ago.

**Pull request merged**